datasketch/experimental/aio/lsh.py

```diff
@@ -186,13 +186,13 @@ def batch_size(self, value):
         """
         await self._insert(key, minhash, check_duplication=check_duplication, buffer=False)
 
-    def insertion_session(self, batch_size=10000):
+    def session(self, batch_size=10000):
```

Comment:
It's better to keep the existing interface `insertion_session` to make it consistent. You can probably create bulk removal in a separate session.

Reply:
OK. I'll redevelop on seperate session. Would you mind if I name it delete_session?

Reply:
Yeah delete_session is good.